.mk/packet.mk

```diff
@@ -47,16 +47,16 @@ packet-stop:
 
 .PHONY: packet-%-load-images
 packet-%-load-images:
-	@if [ -e "scripts/vagrant/images/$*.tar" ]; then \
+	@if [ -e "$(IMAGE_DIR)/$*.tar" ]; then \
 		pushd scripts/terraform; \
 		echo "Loading image $*.tar to master and worker"; \
-		scp ${SSH_OPTS} ../vagrant/images/$*.tar root@`terraform output master${PACKET_CLUSTER_ID}.public_ip`:~/ & \
-		scp ${SSH_OPTS} ../vagrant/images/$*.tar root@`terraform output worker${PACKET_CLUSTER_ID}_1.public_ip`:~/ 
+		scp ${SSH_OPTS} ../../$(IMAGE_DIR)/$*.tar root@`terraform output master${PACKET_CLUSTER_ID}.public_ip`:~/ & \
```

Comment:
Should be here `../$(IMAGE_DIR`instead of  `../../$(IMAGE_DIR)`?

Reply:
Hmm... we are in ./scripts/terraform, and IMAGE_DIR is relative to ./ ... so we'd need to go back up two directories ../../$(IMAGE_DIR) , right?

But it *should* match in the next line down...

Reply:
yep, you are right